src/Tests/Microsoft.NET.Publish.Tests/PublishItemsOutputGroupTests.cs

```diff
@@ -100,6 +100,49 @@ public void GroupNotPopulatedWithoutRid()
             }
         }
 
+        [WindowsOnlyFact]
```

Comment:
I don't think this should be a WindowsOnly fact. Is should run on all platforms where `PublishItemsOutputGroup` is run.

Reply:
PublishItemsOutputGroup is only used by the VS Installer Projects extension (https://marketplace.visualstudio.com/items?itemName=visualstudioclient.MicrosoftVisualStudio2017InstallerProjects), which only runs inside Visual Studio on the Windows platform, so this attribute is appropriate.  I've changed it to include CoreMSBuild as well.

Reply:
Hi @MSLukeWest , I saw some changes where you included diagnostics to test the failures on Linux. Did you figure out what was wrong?  If there's a known reason for that failure, its OK to limit the scenarios, as long as the issue is understood; thanks.